desktopexporter/internal/app/utils/array-to-tree.ts

```diff
@@ -84,8 +84,13 @@ export function arrayToTree(spans: SpanData[]): RootTreeItem[] {
     (a: string, b: string) => {
       let earliestStartTimeA = getEarliestStartTime(lookup[a].children);
       let earliestStartTimeB = getEarliestStartTime(lookup[b].children);
-
-      return earliestStartTimeA - earliestStartTimeB;
+      
+      if (earliestStartTimeA.isBefore(earliestStartTimeB)) {
+        return -1;
+      } else if (earliestStartTimeA.isAfter(earliestStartTimeB)) {
+        return 1;
+      }
+      return 0;
```

Comment:
A `PreciseTimestamp.cmp` function might clean this up, but I kind of like the explicitness here